src/modules/video_player/index.js

```diff
@@ -78,12 +78,58 @@ function togglePlayerCursor(hide) {
     $('body').toggleClass('bttv-hide-player-cursor', hide);
 }
 
+let isMuted = false;
+let isUnloading = false;
+
+function onWindowBlur() {
+    const currentPlayer = twitch.getCurrentPlayer();
+    if (!currentPlayer || !currentPlayer.player || isUnloading) return;
+    isMuted = currentPlayer.player.getMuted();
+    currentPlayer.player.setMuted(true);
+}
+
+function onWindowFocus() {
+    const currentPlayer = twitch.getCurrentPlayer();
+    if (!currentPlayer || !currentPlayer.player) return;
+    currentPlayer.player.setMuted(isMuted);
+}
+
+function onVisibilityChange() {
+    if (document.hidden) {
+        onWindowBlur();
+    } else {
+        onWindowFocus();
+    }
+}
+
+function setUnloading() {
```

Comment:
This is needed because `visibilitychange` triggers when unloading/refreshing the page. It mutes the stream -> twitch updates localstorage and then the stream will always start muted.